lib/Dialect/FIRRTL/Transforms/LowerXMR.cpp

```diff
@@ -283,6 +283,37 @@ class LowerXMRPass : public LowerXMRBase<LowerXMRPass> {
 
   // Propagate the reachable RefSendOp across modules.
   LogicalResult handleInstanceOp(InstanceOp inst) {
+    auto extRefMod =
+        dyn_cast<FExtModuleOp>(instanceGraph->getReferencedModule(inst));
+    if (extRefMod) {
+      // Extern modules can generate RefType ports, they have an attached
+      // attribute which specifies the internal path into the extern module.
+      // This string attribute will be used to generate the final xmr.
+      auto internalPaths = extRefMod.getInternalPaths();
+      if (internalPaths.empty())
+        return success();
```

Comment:
Isn't this actually an error?

Reply:
This is not an error, because if the `internalPaths` is empty, that means no `RefType` ports. So we can just ignore the module.